app/services/nytimes_service.rb

```diff
@@ -10,6 +10,11 @@ def articles
     response_body = parse(connection.get)
     articles_data = response_body["results"]
     Article.create_articles(articles_data)
+    keyword_engine = KeywordEngine.new
+    Article.all.each do |article|
+      keywords = keyword_engine.generate_keywords(article)
+      article.update(keywords: keywords)
```

Comment:
Also should the NYTimes service update keywords? I am sure it's fine. Feel free to make your thoughts known here.